ga4gh/client.py

```diff
@@ -685,11 +686,11 @@ def _run_get_request(self, object_name, protocol_response_class, id_):
             response.text, protocol_response_class)
 
     def _run_list_reference_bases_page_request(self, id_, request):
```

Comment:
If we're using a POST request, then do we need this special casing for list reference bases at all? We should be able to reuse the existing machinery for all the other requests.